src/App.js

```diff
@@ -1,14 +1,38 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
+
 import './App.css';
+//import nav bar and container components for react boostrap
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import Container from 'react-bootstrap/Container';
+
+//import Logo
+import logo from './assets/mglogo4.png';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        Hello React
-      </header>
-    </div>
+    <Navbar bg="light" expand="lg">
+    <Container>
+      <Navbar.Brand href="#home">
+        <img 
+          alt="Logo"
+          src={logo}
+          width="30"
+          height="30"
+          className="d-inline-block align-top"/>{' '}
+        Ingi's Microgreens
+      </Navbar.Brand>
+      <Navbar.Toggle aria-controls="basic-navbar-nav" />
+      <Navbar.Collapse id="basic-navbar-nav">
+        <Nav className="me-auto">
+          <Nav.Link href="#home">Home</Nav.Link>
+          <Nav.Link href="#inventory">Inventory</Nav.Link>
+          <Nav.Link href="#orders">Orders</Nav.Link>
+          <Nav.Link href="#calendar">Calendar</Nav.Link>
+        </Nav>
+      </Navbar.Collapse>
+    </Container>
+  </Navbar>
```

Comment:
This could potentially be refactored (as a part of this pull request or a future one) so that navbar lives in it's own folder/file and is imported. 
That way, each page could independently import the navbar without needing to copy and paste the code. 
Right now, because we don't have multiple pages it's not necessary to abstract the navbar to be it's own component.

Reply:
Good point, for edits do I just change on my local and commit, and re do pull request after?

Reply:
Nevermind I think I understand how the flow works, since its not merged yet with main, looks like it sent up my new commit to the branch here works with existing pull requests :)

Reply:
You got it. 
Typically the decision to include changes as a part of this pull request vs a future pull request has to do with keeping momentum up with adding new code to the project. 
Pretty subjective.